docs/community/governance.md

```diff
@@ -0,0 +1,106 @@
+
+Core Team members always represent the next generation of KOSS and FOSS culture at IIT Kharagpur. They actively volunteer to learn and conduct events, workshops, classes, etc. CTMs also learn how to work in a team. “Community is more important than Code,” and the core team should take this seriously.
+
+At the end of the session, there is a week-long discussion with the Core Team members where they put their ideas forward on topics such as ‘Events,’ ‘Community Standards,’ ‘New hires,’ ‘Meetings and Communications’, etc. After these discussions, the former Executive Heads select the new Executive Heads and Executive Members and terminate the membership of people who are not a cultural fit to work with the Executives.
+
+### 3. Advisors
+
+> Members who have been an Executive before and are still on campus. Late third year onwards.
+
+Advisors, in general, have more technical experience. They actively suggest better technologies to be used in KOSS. They also make sure the Executive Heads are not over-prioritizing anything and are relaxed. They continuously take updates from the Executive Heads about what is going in KOSS. They intervene when asked by an Executive to resolve any conflict. They can also lead meetings if Executive Heads do not object.
+
+One of the Advisors acts as the Treasurer of KOSS and oversees the finances. See [Accounts](/docs/community/accounts).
+
+### 4. Alumni
+
+“Members who have graduated from KGP”
+
+The role has no formal duties, as of now.
+
+
+## Guidelines
+
+### Guidelines for Executive Heads -
+
+1. Do not miss a meeting unless extremely required (health or out of station). Your lack of involvement will psychologically disolve your responsibilities. Schedule meetings according to the availability of all the Executive Heads.
+1. Be in sync among yourselves. Physically meet at-least once a week. Slack and Emails do not take importance over your coordination. You will build an amazing friendship with this, which will remain beyond KOSS.
+1. Respect time of your fellow Executives and Juniors. Do not slack off.
+1. Always have a lump-sum timeline for events and internal workshops for a particular semester (or year). Core Team members are an essential part of overall execution, so not having a timeline puts them at blindside, and they cannot manage their time.
+1. Your Core Team Members are your KGP juniors as well. Make sure they prioritize their mental and physical health.
+1. Follow "Survival first" rule. You need to ensure that your Core Team members have learnt enough to continue in KOSS. You can prioritize this over conducting a fancy workshop.
+1. Make sure KOSS is a sexually and emotionally non-hostile place to work. Importance of this point is definitely not determined by where it is placed.
+1. Build strong connections with your Core Team members. Have regular 1:1s. Prefer Meetings and Calls over Slack and Email.
+1. Keep your Executive Members as involved as possible.
```

Comment:
This is not enough. I think there should be atleast some points on how ?

Reply:
It is very subjective to the batch and their dynamics. I do not know any specific guidelines on the ways of keeping them involved.